src/apps/statements/tests/storingStatements/queriables/getMetadataFromStatement/getNumericQuestionMetadata.test.ts

```diff
@@ -0,0 +1,32 @@
+import * as assert from 'assert';
+
+import { getNumericQuestionMetadata } from '../../../../service/storeStatements/queriables/getMetadataFromStatement/getNumericQuestionMetadata';
+import { numericQuestionInteractionActivityStatement } from './fixtures/numeric-question-interaction.fixture';
```

Comment:
`numericQuestionInteractionActivityStatement` name can be shorter because you already have a module named `numeric-question-interaction.fixture`